app/auth/authenticate_user.rb

```diff
@@ -1,21 +1,21 @@
 class AuthenticateUser
+  def initialize(email, password)
+    @email = email
+    @password = password
+  end
 
-	def initialize(email, password)
-		@email = email
-		@password = password
-	end
+  def call
+    JsonWebToken.encode(user_id: user.id) if user
+  end
 
-	def call
-		JsonWebToken.encode(user_id: user.id) if user
-	end
+  private
 
-	private
+  attr_reader :email, :password
```

Comment:
Hello @thirunjuguna, I need some explanations here. The `attr_accessor` allows `email` and `password` to be instance methods, and we really don't need them to be one in this context; since we are only going to need them from the private methods.

So do you think it's anti-pattern to have private `attr_accessors` or what am I missing in here?